inc/css/class-css-handler.php

```diff
@@ -443,6 +446,28 @@ public static function compress( $css ) {
 		return $css;
 	}
 
+	/**
+	 * Mark in post meta if the post has a review block.
+	 * 
+	 * @param int $post_id Post ID.
+	 * @since 2.4.0
+	 * @access public
+	 */
+	public static function mark_review_block_metadata( $post_id ) {
+		if ( empty( $post_id ) ) {
+			return;
+		}
+
+		$content = get_the_content( '', false, $post_id );
+
+		if ( empty( $content ) ) {
+			return;
+		}
+
+		$has_review = false !== strpos( $content, '<!-- wp:themeisle-blocks/review' );
+		update_post_meta( $post_id, '_themeisle_gutenberg_block_has_review', $has_review );
```

Comment:
We can confirm only to make the change if the current meta value is different than the $has_review variable to avoid adding meta to all the posts where it's not needed, ie adding a false column to posts that don't have a review block to avoid adding useless data to the table.